testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oidc/OIDCDynamicScopeTest.java

```diff
@@ -55,13 +55,13 @@
 @EnableFeature(value = Profile.Feature.DYNAMIC_SCOPES, skipRestart = true)
 public class OIDCDynamicScopeTest extends OIDCScopeTest {
 
-    private static String userId = KeycloakModelUtils.generateId();
+    private static String userId;
 
     @Override
     public void configureTestRealm(RealmRepresentation testRealm) {
         super.configureTestRealm(testRealm);
         UserRepresentation user = UserBuilder.create()
-                .id(userId)
+                .id(KeycloakModelUtils.generateId())
```

Comment:
@hmlnarik Does this line differ then in the class: ResourceOwnerPasswordCredentialsGrantTest.java that id is not deleted but rather generated?